.evergreen/scripts/setfle2parameter.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+"""
+setfle2parameter.py modifies and prints an orchestration config file to add the `--setParameter featureFlagFLE2ProtocolVersion2=1` option to mongod and mongos.
+Usage: setfle2parameter.py <orchestration config file path>
+This file is a temporary workaround until featureFlagFLE2ProtocolVersion2 is enabled by default in SERVER-69563. Once latest server builds have SERVER-69563, this file may be removed.
+DRIVERS-2590 tracks removal of this file.
+"""
+
+import json
+import sys
+import unittest
+import os
+
+
+def do_rewrite(config):
+    did_rewrite = False
+
+    def rewrite_server(server):
+        if "procParams" in server:
+            if "setParameter" in server["procParams"]:
+                server["procParams"]["setParameter"]["featureFlagFLE2ProtocolVersion2"] = "1"
+            else:
+                server["procParams"]["setParameter"] = {
+                    "featureFlagFLE2ProtocolVersion2": "1"
+                }
+            return True
+        return False
+
+    # Rewrite for a server.
+    if rewrite_server(config):
+        did_rewrite = True
+    # Rewrite for each member in a replica set.
+    if "members" in config:
+        for server in config["members"]:
+            if rewrite_server(server):
+                did_rewrite = True
```

Comment:
```suggestion
    members = config.get('members', [])
    mem_rewrites = [rewrite_server(m) for m in members]
    did_rewrite_member = any(mem_rewrites)
```

`Dict.get(K, D)` returns `Dict[K]` if present, otherwise `D`.